app/serializers/aichi11_target_dashboard_serializer.rb

```diff
@@ -83,20 +83,25 @@ def serialize_options
 
   def sorted
     obj_type = @search_type == 'id' ? "AND obj_type = 'country'" : ''
+    default_order = 'ORDER BY region_id, obj_type DESC, name'
```

Comment:
Should this be the region name rather than `region_id`?? Just because I'm getting `Asia` before `Africa` in the default ordering

Reply:
Thanks @lucacug!
I've fixed it now. You should link this up to [this pp-db PR](https://github.com/unepwcmc/protectedplanet-db/pull/24) now though (or merge that one first)